conf/application.conf

```diff
@@ -174,10 +174,11 @@ microservice {
     }
 
     business-details {
-        host = localhost
+        host = 127.0.0.1
         port = 9668
         env = none
         token = secret
+        environmentHeaders = ["Accept", "Gov-Test-Scenario", "Location", "X-Request-Timestamp", "X-Session-Id"]
```

Comment:
should this not be 'environmentHeaders = ["Accept", "Gov-Test-Scenario", "Content-Type", "Location", "X-Request-Timestamp", "X-Session-Id"]'? I might be wrong on this one